aiagents/crew/crew.py

```diff
@@ -30,6 +31,25 @@ def StartCrew(configuration: Initialize):
 
     tasks = Tasks(configuration=configuration, agents=agent_dict)
 
+    embedding = {
```

Comment:
does this work? i didnt know inline if else was allowed in python

Reply:
yes, it is allowed in Python. This part works.